doc/dev/release.md

```diff
@@ -165,20 +182,18 @@ Create a new branch to push release commits:
 $ git checkout -b release-v1.3.0
 ```
 
-Commit changes to the following files:
+Commit the following 9 changes:
```

Comment:
How important is it to include the number of changes? I could see this getting forgotten when we add or remove a step later? Ditto for line 230

Reply:
True, don’t want to add to doc drift.